permiso/src/main/java/com/greysonparrelli/permiso/PermisoDialogFragment.java

```diff
@@ -48,16 +55,102 @@ public static PermisoDialogFragment newInstance(@Nullable String title, @NonNull
         return dialogFragment;
```

Comment:
It'd be great if this version of ```newInstance()``` would just call the other (soon-to-be-named) ```newInstance()```. Then this could just be:

```java
return newInstance(new Builder()
    .setTitle(title)
    .setMessage(message)
    .setButtonText(buttonText));
```